ambry-api/src/test/java/com/github/ambry/account/AccountContainerTest.java

```diff
@@ -888,6 +837,11 @@ private void assertContainer(Container container, int index) {
             container.getSnapshotVersion());
         assertEquals("Wrong accessControlAllowOrigin", refAccessControlAllowOriginValues.get(index),
             container.getAccessControlAllowOrigin());
+        assertEquals("Wrong cacheTtlInSeconds", refContainerCacheTtlInSeconds.get(index),
+            container.getCacheTtlInSecond());
+        assertEquals("Wrong user metadata keys to not prefix in response",
+            refContainerUserMetadataKeysToNotPrefixInResponses.get(index),
```

Comment:
Same as refContainerContentTypeAllowListForFilenamesOnDownloadValues, if it's null, should return an empty set.

Reply:
you are right, updated.